src/main/java/com/o19s/es/ltr/query/LtrQueryBuilder.java

```diff
@@ -130,7 +130,7 @@ protected Query doToQuery(QueryShardContext context) throws IOException {
         }
         features = Collections.unmodifiableList(features);
 
-        RankLibScriptEngine.RankLibModelContainer.Factory factory = context.getScriptService().compile(_rankLibScript, new ScriptContext<>(
+        RankLibScriptEngine.RankLibModelContainer.Factory factory = context.compile(_rankLibScript, new ScriptContext<>(
```

Comment:
nit: (unrelated to this patch so feel free to ignore) I think we could have reused RankLibScriptEngine.CONTEXT instead of rebuilding it.

Reply:
First, thanks for the review! Second, I trust your understanding of the plugin more than my own, in your opinion should this become a separate issue?

Reply:
Thank you both for testing and reviewing my PR :slightly_smiling_face:

I can easily include that change in this PR. It seems safe (and small) enough not to be worth creating a separate ticket.
All the automated checks pass  and I also checked that it is the only remaining place where we aren't using this constant.

Reply:
Wonderful, just push that change and we can merge. Thanks again for contributing this, we really appreciate it!

Reply:
Done :slightly_smiling_face: